doc/operations/recovery.rst

```diff
@@ -30,8 +30,27 @@ To initiate the first phase of the recovery procedure, one or several nodes shou
     recover
     --network-cert-file /path/to/network_certificate
 
+.. mermaid::
+
+    graph LR;
+        Uninitialized-- config -->Initialized;
```

Comment:
I don't think `Uninitialized` is actually observable as it is called from the original ECALL (in fact looking at the code, the entire `node->initialize` call could be merged in the `NodeState` constructor).

Reply:
True, but the state exists internally, so I'm inclined to keep it for now until we've closed #857.